sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -308,12 +308,12 @@ class Analyzer(
           case _ => s
         }
         case m @ Multiply(l, r) if m.childrenResolved => (l.dataType, r.dataType) match {
-          case (CalendarIntervalType, _) => MultiplyInterval(l, r)
-          case (_, CalendarIntervalType) => MultiplyInterval(r, l)
+          case (CalendarIntervalType, _) => MultiplyInterval(l, r, conf.ansiEnabled)
```

Comment:
`SQLConf.get` is reliable now. Let's follow other PRs and use the default parameter value.